packages/liveblocks-react-comments/src/primitives/Comment/index.tsx

```diff
@@ -80,6 +94,19 @@ const CommentBody = forwardRef<HTMLDivElement, CommentBodyProps>(
                       ) : null;
                     }
 
+                    if (isComposerBodyAutoLink(inline)) {
+                      return (
+                        <AutoLink
+                          href={toAbsoluteURL(inline.href)}
+                          target="_blank"
+                          rel="noopener noreferrer nofollow"
+                          key={index}
```

Comment:
Do we want to provide styling option via `className` or the render prop similar to the mention component? @marcbouchenoire

Reply:
Via `renderLink` (which would be the same render prop for both auto and later custom links), given that this is in a primitive where we should always give access to the markup if possible (contrary to the default components).